### ISSUE TITLE
ctc_loss_calculator.cc Not a valid path

### ISSUE THREAD

**Issue by Parshwa27** [weinman/cnn_lstm_ctc_ocr#52]
While training OCR I got the following error a couple of times only:
Error: W tensorflow/core/util/ctc/ctc_loss_calculator.cc:144] No valid path found
I don't understand what it means. Can anyone help?

**Comment by weinman** [comment]
Either a learned probability for a ground truth label sequence [has collapsed to zero](https://github.com/tensorflow/tensorflow/blob/444a8bc05f73d820c6146c914759fda36240a34c/tensorflow/core/util/ctc/ctc_loss_calculator.cc#L144) or your ground truth label sequence is [longer than the feature sequence](https://github.com/tensorflow/tensorflow/blob/444a8bc05f73d820c6146c914759fda36240a34c/tensorflow/core/util/ctc/ctc_loss_calculator.cc#L43). 
See https://stackoverflow.com/questions/45130184/ctc-loss-error-no-valid-path-found

**Comment by Parshwa27** [comment]
Is this the reason that the model is omitting every double character? E.g. "ned" instead of "need".

**Comment by weinman** [comment]
@Parshwa27 Not likely. More likely is that the model has not yet learned to insert the CTC blank character needed to prevent the repeated characters from being merged during the beam search.
